### PR TITLE
Clears out my old donor items + makes the drawing tablet available to all donors

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Exactly what the title says, its going to take me a hot minute to figure out how to code in my new items so in the meantime I'm taking out the proverbial trash. 

## How This Contributes To The Nova Sector Roleplay Experience

Me being able to larp is fantastic , opening 5 slots for me to larp harder is more epic

## Proof of Testing
Code work good still

code work good

## Changelog

:cl: Thedragmeme
add: Gives all donators the ability to use the drawing tablet
del: My old donor items I don't use anymore
/:cl:

### CODE REVIEW

**Comment by CandleJaxx** [comment]
woah, are we able to replace old donor items again? i thought they stopped that practice.. based...

File: modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -96,8 +96,9 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
 	supports_variations_flags = NONE
 
-// Donation reward for Thedragmeme
+// Donation reward
 // might make it have some flavour functionality in future, a'la rewritable piece of paper - JOKES ON YOU I'M MAKING IT DRAW
+// Note from thedragmeme- The fact this can actually draw is epic, Im making this an item accessable to all donors so if an actual coder could make it print out drawings or make it save the drawings to the library that would be epic
 /obj/item/canvas/drawingtablet
 	name = "drawing tablet"
 	desc = "A portable tablet that allows you to draw. Legends say these can earn the owner a fortune in some sectors of space."
```
Comment (Draganfrukts) [inline modular_nova/modules/customization/modules/clothing/~donator/donator_clothing.dm:104]: real

**Comment by Draggeru** [comment]
> woah, are we able to replace old donor items again? i thought they stopped that practice.. based...

I spoke with Golden and got permission, since my items are from 2019/2020

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.

**Pull request merged**